src/components/Thanks/ThanksPageSingleVersion.vue

```diff
@@ -72,10 +102,18 @@ const props = defineProps({
 		type: Array,
 		default: () => ([]),
 	},
+	router: {
```

Comment:
Should we use `useRouter` composable instead of passing router function? Sometimes an anti-pattern in vue...

Reply:
Oh yup, good call. I was following the old Vue 2 pattern in the TY page parent component. Fixed!